_layouts/default.html

```diff
@@ -18,7 +18,7 @@
             {% if language[0] == "en" %}
               href="/"
             {% else %}
-              href="{{ language[0] | prepend: "/lang/" }}"
+              href="{{ language[0] | prepend: '/lang/' | append: '/' }}"
```

Comment:
```suggestion
              href="{{ language[0] | prepend: "/lang/" | append: "/" }}"
```

Let's be consistent

Reply:
@alexandrtovmach but quote in quote will be dangerous:

```
href="..."xxx"..."
```

Reply:
It's not "quote in quote" actually. Everything inside `{{...}}` it's expression, so should be fine. At least, it was implemented in the same way before